### ISSUE TITLE
Version gets updated wrongly

### ISSUE THREAD

**Issue by bjornbouetsmith** [vers-one/dotnet-project-version-updater#2]
Hi,

I have a project (Web API project)
~~~xml
<Project Sdk="Microsoft.NET.Sdk.Web">
  <PropertyGroup>
    <Version>1.0.1</Version>
    <TargetFramework>net6.0</TargetFramework>
    <UserSecretsId>1b94c50a-02c2-443a-82b2-9e0443a56993</UserSecretsId>
    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
    <GenerateDocumentationFile>False</GenerateDocumentationFile>
    <AssemblyVersion>$(Version)</AssemblyVersion>
    <FileVersion>$(Version)</FileVersion>
  </PropertyGroup>
~~~

When I run this as part of a github action:
~~~yaml
    - name: Update version
      id: update_version
      uses: vers-one/dotnet-project-version-updater@v1.0
      with:
        file: Source/Api/Api.csproj
        version: bump-build
~~~

It logs:

~~~
Run vers-one/dotnet-project-version-updater@v1.0
  with:
    file: Source/Api/Api.csproj
    version: bump-build
  env:
    DOTNET_ROOT: /home/runner/.dotnet
File path: Source/Api/Api.csproj
New version: bump-build
Version updated from 1.0.1 to 1
~~~

Which obviously is wrong - is this because its a `Microsoft.NET.Sdk.Web`project and not a `Microsoft.NET.Sdk` project - and is this something you can fix?

**Comment by vers-one** [comment]
Reproduced. I will make a fix by the end of this week.

**Comment by vers-one** [comment]
Thank you for reporting this! The fix is available in v1.1.
